link between fraud and unblock

richagi opened the pull request:
Clarified in the fraud section how an admin can unblock a users after they report fraud.

azurecla commented:
Hi __@jeffhollan__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

deneha commented:
CLA not required.  User is MSFT employee.

richagi committed:
link between fraud and unblock

Clarified in the fraud section how an admin can unblock a users after they report fraud.
